### ISSUE TITLE
Uncaught TypeError: Cannot read property 'update' of null

### ISSUE THREAD

**Issue by AnubisCode** [orange-games/phaser-input#41]
just start your example and got error:
```
Uncaught TypeError: Cannot read property 'update' of null
    at c.update (phaser-input.min.js:10)
    at c.World.c.Group.update (phaser.js:33716)
    at c.Stage.update (phaser.js:31820)
    at c.Game.updateLogic (phaser.js:36338)
    at c.Game.update (phaser.js:36280)
    at c.RequestAnimationFrame.updateRAF (phaser.js:61979)
    at window.requestAnimationFrame.forceSetTimeOut._onLoop (phaser.js:61962)
```

**Comment by lootwinner** [comment]
Same, realised the error,
update() is calling this.placeholder.update() without checking if this.placeholder is null..
I just adding the check to the .js file, but hopefully it gets fixed soon.

**Comment by lootwinner** [comment]
PR created https://github.com/orange-games/phaser-input/pull/42

**Comment by AleBles** [comment]
Thanks, I'll be updating to a new version soon :)

**Comment by Tdnshah** [comment]
I am using phaser input and got the error on Uncaught TypeError: Cannot read property 'update' of null ' with this.placeHolder and as on the issue #41 @CFix-Ollie suggested to add a check on this.placeHolder before its called by update and also he gave a commit for the phaser-input.ts file. I faced the same problem in javascript version and the same solution worked for me too hence I have committed same changes for phaser-input.js.

i haven't committed any changes to phaser-input.min.js hence if any one using minified then it still needs to be updated.
